jquery.i18n/languages/ru.json

```diff
@@ -0,0 +1,10 @@
+{
+	"@metadata": {
+    "authors": ["JustinCast"],
+    "last-updated": "2020-03-20",
+    "locale": "ru"
+  },
+  "sub-heading-1": "RUSIAN GOES HERE",
```

Comment:
@JustinCast 

Let's add all keys for this JSON  , they can have blank values for now but It would be very helpful for the rusian translation